src/main/java/ToDo.java

```diff
@@ -0,0 +1,11 @@
+public class ToDo extends Task {
+
+    public ToDo(String description) {
+        super(description);
+    }
+
+    @Override
+    public String toString() {
+        return "[T]" + super.toString();
+    }
```

Comment:
Good use of overloading the toString method

Reply:
Thank you!